index.html

```diff
@@ -456,8 +456,8 @@
 		<div class="col-sm-2">Trung Huynh&nbsp;&nbsp;<i class="fa fa-rocket"></i></div>
 		<div class="col-sm-2">
 			<a href="https://github.com/tkhuynh" target="_blank"><i class="fa fa-github-square fa-2"></i></a> &nbsp;&nbsp;
-			<i class="fa fa-linkedin fa-2"></i> &nbsp;&nbsp;
-			<i class="fa fa-twitter fa-2"></i>&nbsp;&nbsp;
+			<a href="https://www.linkedin.com/in/trunghuynh1" target="_blank"><i class="fa fa-linkedin fa-2"></i> &nbsp;&nbsp;
```

Comment:
Please add a closing </a> tag after the closing </i> tag so that the icon (and nothing else) is hyperlinked.